Vocable/Features/Settings/SelectionModeCollectionViewController.swift

```diff
@@ -0,0 +1,88 @@
+        let layout = createLayout()
+        collectionView.collectionViewLayout = layout
+    }
+    
+    func createLayout() -> UICollectionViewLayout {
+        if case .compact = self.traitCollection.verticalSizeClass {
+            return compactVerticalLayout()
+        } else {
+            return regularHeightWidthLayout()
+        }
+    }
+    
+    private func compactVerticalLayout() -> UICollectionViewLayout {
+        let headTrackingToggleItem = NSCollectionLayoutItem(layoutSize: NSCollectionLayoutSize(widthDimension: .fractionalWidth(1.0), heightDimension: .fractionalHeight(1.0)))
+        let headTrackingToggleGroupSize = NSCollectionLayoutSize(widthDimension: .fractionalWidth(1.0), heightDimension: .fractionalHeight(1/5))
+        let headTrackingToggleGroup = NSCollectionLayoutGroup.vertical(layoutSize: headTrackingToggleGroupSize, subitems: [headTrackingToggleItem])
+        
+        let section = NSCollectionLayoutSection(group: headTrackingToggleGroup)
+        let layout = UICollectionViewCompositionalLayout(section: section)
+        return layout
+    }
+    
+    private func regularHeightWidthLayout() -> UICollectionViewLayout {
+        let headTrackingToggleItem = NSCollectionLayoutItem(layoutSize: NSCollectionLayoutSize(widthDimension: .fractionalWidth(1.0), heightDimension: .fractionalHeight(1.0)))
+        let headTrackingToggleGroupSize = NSCollectionLayoutSize(widthDimension: .fractionalWidth(1.0), heightDimension: .fractionalHeight(1/9))
+        let headTrackingToggleGroup = NSCollectionLayoutGroup.vertical(layoutSize: headTrackingToggleGroupSize, subitems: [headTrackingToggleItem])
+        
+        let section = NSCollectionLayoutSection(group: headTrackingToggleGroup)
+        let layout = UICollectionViewCompositionalLayout(section: section)
+        return layout
+    }
+
+    override func numberOfSections(in collectionView: UICollectionView) -> Int {
+        return 1
+    }
+
+    override func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return 1
+    }
+    
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+        collectionView.setCollectionViewLayout(createLayout(), animated: false)
+    }
+    
+    override func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+        if collectionView.indexPathForGazedItem != indexPath {
+            collectionView.deselectItem(at: indexPath, animated: true)
+        }
+        
+        if AppConfig.isHeadTrackingEnabled {
+            let alertViewController = GazeableAlertViewController.make { AppConfig.isHeadTrackingEnabled.toggle() }
+            present(alertViewController, animated: true)
+            alertViewController.setAlertTitle("Turn off head tracking?")
+        } else {
+            AppConfig.isHeadTrackingEnabled.toggle()
+        }
```

Comment:
```suggestion
        let alertViewController = GazeableAlertViewController.make { AppConfig.isHeadTrackingEnabled.toggle() }
        present(alertViewController, animated: true)
        alertViewController.setAlertTitle("Turn off head tracking?")
```
The cell should not be selectable if head tracking is disabled. Check the implementation from the other view controller for the delegate methods to override for that. I believe highlight and selection were mitigated via delegate methods.